test/tools/WebListener/Controllers/GetController.cs

```diff
@@ -17,13 +17,13 @@ public JsonResult Index()
             Hashtable args = new Hashtable();
             foreach (var key in Request.Query.Keys)
             {
-                args.Add(key, string.Join(Constants.HeaderSeparator, Request.Query[key]));
+                args.Add(key, string.Join(Constants.HeaderSeparator, (string)Request.Query[key]));
```

Comment:
Could you please report to Runtime repo the issue?

I see there are:
https://github.com/dotnet/runtime/issues/101261
https://github.com/dotnet/runtime/pull/101499
https://github.com/dotnet/runtime/pull/101457

@xtqqczze I suggest to revert to 11.0 lang for the test project so that we can merge and don't wait comments from dotnet team.